patient-care/webapp/client/shared/editSampleGroup.js

```diff
@@ -335,8 +336,10 @@ Template.formValuesFilter.onCreated(function(){
   instance.editing = new ReactiveVar(false);
 
   let dataset_id = instance.data.data_set_id ;
-  instance.available_filter_forms = new ReactiveVar(); 
-  instance.available_filter_forms.set([{name: "Loading forms...", formId: "placeholder_loadingforms"}]);
+  instance.available_filter_forms = new ReactiveVar();
+  instance.available_filter_forms.set([{name: "Loading forms...", urlencodedId: "placeholder_loadingforms"}]);
```

Comment:
Yeah, merge in all lost changes from  https://github.com/UCSC-MedBook/MedBook_/pull/29/files/a099dfa926a60d728d31b77944fefe324f844ab6..16dcfe79807581c8657807b64cb00c4a87d4eb93 first & then after that, worry about what other pulls have gotten lost.